adapters/MoPub/Public/Headers/MoPubAdapter.h

```diff
@@ -1,5 +1,3 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
-/// Import all the public headers here.
-//#import <[AD_NETWORK_NAME]Adapter/[YOUR_PUBLIC_HEADERS].h>
```

Comment:
Need to import `GADMoPubNetworkExtras` as follows:
`#import <MoPubAdapter/GADMoPubNetworkExtras.h>`